configure.ac

```diff
@@ -396,6 +399,35 @@ AS_IF([test "x$enable_shared" = "xyes"], [
     #  PowerPC and SPARC.
     # Therfore use -fPIC for now
     CXXFLAGS="$CXXFLAGS -fPIC"
+    # for make.config
+    DEFLIB1='$(LIB_SO)'
+    # for src/makefile
+    DEFLIB2+=' ../lib/libbout++.so'
+    AS_IF([test "x$enable_static" = "xauto"], [
+        enable_static=no
+    ])
+    SHARED_EXTRA=':'
+    AS_IF([test "x$enable_static" = "xno"], [
+        SHARED_EXTRA='$(RM) -f ../lib/*.a'
+    ])
+], [
+    AS_IF([test "x$enable_static" = "xauto"], [
+        enable_static=yes
+    ])
+])
+
+AS_IF([test "x$enable_static" = "xyes"], [
+    DEFLIB1='$(LIB_A)'
```

Comment:
What happens if both `--enable-shared --enable-static` are passed?

Reply:
both are build.
A rebuild is triggered if the static is determined out of date. But unless you e.g. delete the `.so` both should always be updated.

Reply:
:+1: It just looked like `DEFLIB1` is being clobbered here so I was worried that it might not build the shared lib too, but it looks like it all works great!